ui/src/app/app-router.tsx

```diff
@@ -79,12 +79,12 @@ export const AppRouter = ({popupManager, history, notificationsManager}: {popupM
                         navItems={[
                             {
                                 title: 'Workflows',
-                                path: workflowsUrl + '/' + namespace,
+                                path: workflowsUrl + (Utils.managedNamespace ? '' : '/' + namespace),
```

Comment:
I think you can simplify this to `Utils.getNamespace(namespace)`?

Reply:
So, this is actually blanking out the namespace entirely. The goal is to just have it point to /workflows rather than /workflows/namespace. I think, ideally, in managed namespace mode, you don't even need to be aware of the NS. This is how it went in V2, as well.

What say you?